restli-tools/src/main/java/com/linkedin/restli/tools/clientgen/RestRequestBuilderGenerator.java

```diff
@@ -51,8 +51,8 @@
  */
```

Comment:
Can you add a `TODO` here which says that we want to refactor this tool into a nicer CLI app using Commons-CLI?

Reply:
Done.